src/components/composite/sections/ContextSection.tsx

```diff
@@ -2,16 +2,18 @@ import MenuBookIcon from '@mui/icons-material/MenuBook';
 import PeopleAltIcon from '@mui/icons-material/PeopleAlt';
```

Comment:
How is this change related to the story?

Reply:
It's not, but it was initially. Most of the work I did before Aspects were moved from Context to Dashboard. Should I revert changes to the `ContextSection`? (I mean, they look as an improvement to me).

Reply:
They are an improvement - don't revert them, but also-  where do you draw the line?